tests/test_style_cases.py

```diff
@@ -49,7 +49,10 @@ def _load_test_cases():
 def _checker(filename, tree, style_entry_point):
     options = {
         'application_import_names': [
-            'flake8_import_order', 'namespace.package_b', 'tests',
+            'namespace.package_b',
```

Comment:
I think we'd want separate tests for this. We'd want to make sure the old version works, as well as the new application paths work. I think we'd even want to ensure they do exactly the same thing.